selectors.yml

```
@@ -130,6 +133,9 @@ selectors:
   255:
     selector: 3719320017875267166
     name: "ethereum-mainnet-kroma-1"
+  1088:
+    selector: 8805746078405598895
+    name: "ethereum-mainnet-andromeda-metis-1"
```

What's this? Is the network named `andromeda-metis`? What are each part of this name?

---

From what I can see, their mainnet is called `Metis Andromeda` in internal docs and Chainlist, but I don't see signs of another instance of this rollup, and everywhere else it's called only Metis. So maybe this would be better as:
```suggestion
    name: "ethereum-mainnet-metis-1"
```

---

https://docs.metis.io/dev/readme/connection-details

"andromeda" is what they call their mainnet. That is the network_name. Do we only include network_name for testnets?

---

Yeah, I thought that as well. Including network_name doesn't make much sense to me for mainnets.